hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestBucketManagerImpl.java

```diff
@@ -67,51 +85,51 @@ private OzoneConfiguration createNewTestPath() throws IOException {
     return conf;
   }
 
-  private OmMetadataManagerImpl createSampleVol() throws IOException {
+  private void createSampleVol() throws IOException, AuthenticationException {
     OzoneConfiguration conf = createNewTestPath();
-    OmMetadataManagerImpl metaMgr = new OmMetadataManagerImpl(conf);
-    String volumeKey = metaMgr.getVolumeKey("sampleVol");
+    omTestManagers = new OmTestManagers(conf);
+    writeClient = omTestManagers.getWriteClient();
+
     // This is a simple hack for testing, we just test if the volume via a
     // null check, do not parse the value part. So just write some dummy value.
     OmVolumeArgs args =
         OmVolumeArgs.newBuilder()
-            .setVolume("sampleVol")
+            .setVolume("sample-vol")
             .setAdminName("bilbo")
             .setOwnerName("bilbo")
             .build();
-    metaMgr.getVolumeTable().put(volumeKey, args);
-    return metaMgr;
+    writeClient.createVolume(args);
+
   }
 
   @Test
   public void testCreateBucketWithoutVolume() throws Exception {
     thrown.expectMessage("Volume doesn't exist");
     OzoneConfiguration conf = createNewTestPath();
-    OmMetadataManagerImpl metaMgr =
-        new OmMetadataManagerImpl(conf);
+    omTestManagers = new OmTestManagers(conf);
+
     try {
-      BucketManager bucketManager = new BucketManagerImpl(metaMgr);
+      OzoneManagerProtocol writeCl = omTestManagers.getWriteClient();
```

Comment:
we've used writeClient everywhere else.  I would use that instead of writeCl here.

Reply:
I declared OzoneManagerProtocol as well based on your PR and then I was getting an error from checkstyle about hiding the class field with the same name. But now I realize there is a so much better way and simpler way because the class field has not been initialized. I will fix that.